centreon/www/class/centreonLDAP.class.php

```diff
@@ -344,7 +344,7 @@ public function escapeLdapFilterSpecialChars($name): string
      * @param string $username The username
      * @return string|bool The dn string or false if not found
      */
-    public function findUserDn($username): ?string
+    public function findUserDn($username): string|bool
```

Comment:
In PHP 8.2 you can type with true or false

```suggestion
    * @return string|false The dn string or false if not found
    */
    public function findUserDn($username): string|false
```